cvat-ui/src/components/annotation-page/standard3D-workspace/context-image/context-image.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useEffect } from 'react';
+
+interface Props {
+    frame: number;
+    contextImageHide: boolean;
+    loaded: boolean;
+    data: string;
+    getContextImage(): void;
+}
+
+export default function ContextImage(props: Props): JSX.Element {
+    const {
+        frame, contextImageHide, loaded, data, getContextImage,
+    } = props;
+
+    useEffect(() => {
+        if (!contextImageHide && !loaded) {
+            getContextImage();
+        }
+    }, [frame, contextImageHide, loaded]);
+
+    const renderImage = (): JSX.Element => {
+        if (loaded) {
+            if (data === '') {
+                return null;
+            }
+            return <img src={data} alt='' style={{ maxWidth: '100%', maxHeight: '100%' }} />;
+        }
+
+        return <div style={{ textAlign: 'center' }}> Loading</div>;
```

Comment:
**style** doesn't contains any dynamic variables, so, please use the css class in places like this